proto/stat_var.proto

```diff
@@ -180,16 +181,16 @@ message StatVarGroupNode {
 }
 
 message GetStatVarGroupRequest {
-  repeated string places = 2;
+  repeated string entities = 2;
   reserved 1;
 }
 message GetStatVarGroupNodeRequest {
   // The stat var or stat var group ID to query for.
   string stat_var_group = 1;
-  // The places that the stat var group is associated with. The response should
-  // only contain the children stat var (group) if it has data for at least one
-  // of the places.
-  repeated string places = 2;
+  // The entities that the stat var group is associated with. The response
+  // should only contain the children stat var (group) if it has data for at
+  // least one of the entities.
+  repeated string entities = 2;
```

Comment:
To confirm, stanford doesn't use this API, right?  They wanted to use some SV hierarchy APIs in the past.

Reply:
They don't have public traffic. This is the requests in the past 30 days https://screenshot.googleplex.com/9kihiGV3wFwYmag

Reply:
Great, thanks for checking!

Reply:
thank you both :)